util/src/main/java/org/datacommons/util/ExistenceChecker.java

```diff
@@ -204,6 +206,19 @@ private void performDcCall(
       if (verbose) {
         logger.info("DC call failed for - " + Strings.join(subs, ',') + ", " + pred);
       }
+      // If this was an independent RPC call, don't want to re-issue the call.
+      if (subs.size() == 1) {
+        var sub = subs.get(0);
+        if (!subMap.containsKey(sub)) {
+          return;
+        }
+        var objMap = subMap.get(sub);
+        for (var cbs : objMap.values()) {
+          totalPendingCallCount -= cbs.size();
+        }
```

Comment:
should we loop over the cbs and do: `cb.logError("Existence_FailedDCCall", "Failed to ...");` or something?

that way we bubble up the error instead of silently dropping.

Reply:
done